tests/functional/object/mcg/test_mcg_performance_profiles.py

```
@@ -0,0 +1,478 @@
+                        "PV pool agent",
+                    ), f"PV pool agent resources do not match '{profile}' profile"
+
+                    logger.info("PV pool agent pod resources verified ✓")
+                else:
+                    logger.warning(
+                        "PV pool backingstore exists but no agent pods found"
+                    )
+            else:
+                logger.info(
+                    f"Backingstore type is '{bs_type}' (cloud storage), "
+                    "skipping PV pool verification (N/A for cloud platforms)"
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)